cpp/ql/test/library-tests/templates/typedefs/template_typedefs.expected

```diff
@@ -12,12 +12,14 @@
 | F<int> | int |
 | G<..(*)(..)> | pointer to {function returning {pointer to {int}} with arguments (int)} |
 | G<TG> | TG |
-| G<__attribute((vector_size(32))) int> | {GNU 8 element vector of {int}} |
+| G<__attribute((vector_size(32))) int> | GNU 8 element vector of {int} |
```

Comment:
Why has this changed?

Reply:
I believe the curly braces represent a typedef that the old extractor failed to strip.

Reply:
:+1: